engines/wazero/wazero.go

```diff
@@ -321,6 +320,10 @@ func (m *Module) Instantiate(ctx context.Context) (wapc.Instance, error) {
 	return &instance, nil
 }
 
+func (i *Instance) RemainingPoints(context.Context) uint64 {
```

Comment:
I would remove this from the top-level interface as then you don't need to document no-op on something only one engine supports.

Reply:
@codefromthecrypt Let me clean this up a bit more.  I see your points and will try to make it so.